Controller/GluggiController.php

```diff
@@ -121,28 +121,42 @@ public function htmlTitle (GluggiConfig $config, $pageTitle)
      * @param GluggiConfig       $config
      * @param AssetHtmlGenerator $htmlGenerator
      * @param string             $type
+     * @param array              $addAssets
+     * @param array              $overrideAssets
+     *
```

Comment:
Remove this empty line please

Reply:
This empty line is made in the whole file to separate `@param` and `@return`

Reply:
Urgs.. okay, leave it like that